src/corehost/cli/deps_resolver.cpp

```diff
@@ -486,6 +486,16 @@ void deps_resolver_t::resolve_probe_dirs(
     {
         if (probe_entry_in_configs(entry, &candidate))
         {
+            // For standalone apps, on win7, coreclr needs ApiSets which has to be in the DLL search path.
+            if (!m_portable && pal::need_api_sets()
```

Comment:
Ideally, all native dirs will be added, this will make that worry go away. We decided to make a point fix because we don't expect this to change. Besides, hostpolicy is the most serviceable component and corehost the least.